src/render-html/js.js

```diff
@@ -102,6 +102,12 @@ const handleLongPress = e => {
   if (!lastTouchEventTimestamp || Date.now() - lastTouchEventTimestamp < 500) return;
 
   lastTouchEventTimestamp = 0;
+
+  sendMessage({
```

Comment:
Now long pressing any element in webView, event is sent.
If it is message node then event contains `target: 'message'` & `messageId`.
If it is header then event contains `target: 'header'` & `messageId`.
For all other events `messageId` is 0, thus that event is discarded at the time of handling it (in `webViewEventHandler.js`). 

Before this only long press message content & header, event is sent. I was thinking to only consider long press on message content. But ya! full message node needs to be target.

Reply:
Cool! That looks good!
If we need to handle more logic and it can be handled in both the webview and native app, prefer the native app, so we can test/debug it more easily.